iothub/service/src/JobClient/JobClient.cs

```diff
@@ -1,32 +1,54 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Common;
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Devices.Common.Exceptions;
+using System.Net.Http;
+using System.Text;
 
 namespace Microsoft.Azure.Devices
 {
-    using Microsoft.Azure.Devices.Shared;
-    using System;
-    using System.Threading;
-    using System.Threading.Tasks;
-
     /// <summary>
     /// Job management
     /// </summary>
-    public abstract class JobClient : IDisposable
+    public class JobClient : IDisposable
```

Comment:
No longer abstract and has all the implementation here so all the virtual methods below can be mocked easily.